docs/source/training_pipeline/01_getting_started.rst

```diff
@@ -0,0 +1,279 @@
+.. include:: /include/substitution.rst
+
+********
+Overview
+********
+
+==============
+ Introduction 
+==============
+
+This section will introduce you to the training pipeline, its features, how it works, how you can configure the parameters and how you can train on your own dataset.
+PeekingDuck offers this feature for users who intend to perform their own model training.
+Training is possible without the need to write boilerplate codes. The training pipeline supports the image classification use case as of PeekingDuck version 2.0.
+
+
+Features
+--------
+
+.. raw:: html
+
+   <h6>TensorFlow and PyTorch Models</h6>
+Supports pre-trained models from both PyTorch and TensorFlow. The selection of the model can be customized in the configuration files.
```

Comment:
For `Features` section, need to standardize on the sentence structure style. Above point uses clauses (partial sentences), e.g. "Supports pre-trained models ...", but `Analyze Runs` point below uses full sentences.
As this is _not_ a marketing document, we should write in complete sentences.

Reply:
Well noted. Will update to the full sentences.